### ISSUE TITLE
is pinchZoomFactor() on android chrome supposed to work?

### ISSUE THREAD

**Issue by AndrewJDR** [hashchange/jquery.documentsize#5]
I tried the following code with Chrome on Android and the pinchZoomFactor() always returns 1. Is this expected/normal?

```
<html>
   <head>
   </head>
<body>
   <div>
      Page Zoom: ...
   </div>
</body>
<script>
let $ = {};
</script>
<script src="jquery.documentsize.js">
</script>
<script>
document.addEventListener("touchend", () =>
      {
         setTimeout(() => {
            document.body.innerHTML = "Zoom level: " + $.pinchZoomFactor().toString();
         }, 500);
      }, false);
</script>
</html>
```

**Comment by hashchange** [comment]
Sorry for having let you wait that long for an answer. `pinchZoomFactor()` is supposed to work with Chrome on Android, but you are right, it is broken now.

The Chrome team [had already tinkered with](https://www.quirksmode.org/blog/archives/2016/02/chrome_change_b.html) `window.innerWidth` and `window.innerHeight` last year, a well-intentioned but still disastrous decision in my opinion, before [reverting the change](https://bugs.chromium.org/p/chromium/issues/detail?id=571297#c95). Turns out they've [done it again this year](https://www.quirksmode.org/blog/archives/2017/09/chrome_breaks_v.html).

The difference between this year and last is that they've also put a new API in place, which should allow me to get things working again. I still think it is disastrous for the Chrome team to second-guess usage on the web as a whole, and modify the behaviour of methods which have worked consistently across all browsers for ages. Instead of getting rid of the confusion around viewport dimensions, they've just added to it. A second, independent API for the layout viewport, [as ppk has suggested](https://www.quirksmode.org/blog/archives/2017/09/chrome_breaks_v.html#link5), would have been a much more helpful move. But given the market share they command, they do get away with decisions like that.

Anyway, I'll try and get a fix in place for Chrome on Android as soon as I can.

**Comment by hashchange** [comment]
Fixed in 1.2.4.
